wchat/screens/shared/components/ContactTable/index.js

```diff
@@ -12,9 +12,12 @@ export default class ContactTable extends Component {
       id={item.id}
       title={item.username}
       avatarUrl={item.avatar}
-      onPress={()=>{this.props.onSelect(item.id)}}
+      onPress={()=>this.onSelect(item.id)}
```

Comment:
mmm im sorry, in this case leave the `this.props.onSelect(item.id)` because we are in the same problem that before.